bitfinex-client/src/client/mod.rs

```diff
@@ -349,6 +350,66 @@ impl BitfinexClient {
         Ok(transfer)
     }
 
+    #[instrument(name = "bitfinex_client.funding_account_balance", skip(self), err)]
+    pub async fn funding_account_balance(
+        &self,
+    ) -> Result<FundingAccountBalance, BitfinexClientError> {
+        let mut body: HashMap<String, String> = HashMap::new();
+        body.insert("type".to_string(), Wallet::MARGIN.to_string());
+        if self.config.simulated {
+            body.insert("symbol".to_string(), "fUSD".to_string());
+        } else {
+            body.insert("symbol".to_string(), Currency::USD.to_string());
+        }
+        let request_body = serde_json::to_string(&body)?;
+
+        let endpoint = "/order/avail";
+        let params = "";
+        let headers = self.post_calc_request_headers(endpoint, params, &request_body)?;
+
+        let response = self
+            .rate_limit_client(endpoint)
+            .await
+            .post(Self::url_for_auth_calc_path(endpoint, params))
+            .headers(headers)
+            .body(request_body)
+            .send()
+            .await?;
+        let funding_balance =
+            Self::extract_response_data::<FundingAccountBalance>(response).await?;
+        Ok(funding_balance)
+    }
+
+    #[instrument(name = "bitfinex_client.trading_account_balance", skip(self), err)]
+    pub async fn trading_account_balance(
+        &self,
+    ) -> Result<TradingAccountBalance, BitfinexClientError> {
+        let mut body: HashMap<String, String> = HashMap::new();
+        body.insert("type".to_string(), Wallet::EXCHANGE.to_string());
+        if self.config.simulated {
+            body.insert("symbol".to_string(), "fUSD".to_string());
+        } else {
+            body.insert("symbol".to_string(), Currency::USD.to_string());
```

Comment:
@sebastienverreault what is the right value for symbol. i was not sure so used ```fUSD``` to test the implementation.

Reply:
Aren't "fUSD" and Currency::USD the same?
We probably want the balance of the accounts in Currency::BTC, right?
As USD is the exposure we get by shorting the BTC/USD SWAP instrument based on some BTC posted margin.

Reply:
Also the symbol usually refers to an instrument rather than a currency
so it looks like the endpoint is not the right one to get the balance of the account.

The doc says: "Calculate the balance available for orders/offers"
so sounds like it would give a number of contracts, for ex., that you can buy/sell
rather than the balance of the account in BTC or else.

Reply:
You are right probably... I think https://docs.bitfinex.com/reference/rest-auth-info-margin this is the correct endpoint